tee-worker/cli/test_auto_shielding_with_transfer_bob.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+set -euo pipefail
+
+# Verifies that auto shielding transfers sent to vault account: //Alice are verified from sender //Bob
+#
+
+while getopts ":m:p:A:u:V:w:x:y:z:C:" opt; do
+    case $opt in
+        p)
+            INTEGRITEE_RPC_PORT=$OPTARG
+            ;;
+        A)
+            WORKER_1_PORT=$OPTARG
+            ;;
+        u)
+            INTEGRITEE_RPC_URL=$OPTARG
+            ;;
+        V)
+            WORKER_1_URL=$OPTARG
+            ;;
+        w)
+            TARGET_A_PARENTCHAIN_RPC_URL=$OPTARG
+            ;;
+        x)
+            TARGET_A_PARENTCHAIN_RPC_PORT=$OPTARG
+            ;;
+        C)
+            CLIENT_BIN=$OPTARG
+            ;;
+        *)
+            echo "invalid arg ${OPTARG}"
+            exit 1
+    esac
+done
+
+# Using default port if none given as arguments.
+INTEGRITEE_RPC_PORT=${INTEGRITEE_RPC_PORT:-9944}
```

Comment:
LITENTRY ?

Reply:
This file is not used in tests either, so I didn't bother changing it